README.md

```diff
@@ -1 +1,190 @@
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Set up Hello Microverse project] <a name="about-project"></a>
+
+In this project, I set up a "Hello world" repository. No complex coding is done for this Project. My goal here is to master all of the tools and best practices I learned so far  about software development. 
+
+## 🛠 Built With <a name="built-with"></a>
+1.css
+2.HTML
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+  <ul>
+     <li>Linters</li>
+    <li>CSS3</li>
+    <li>HTML5</li>
+  </ul>
+
+
+
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **Added  index.html file**
+- **Added style.css file**
+- **Added linter file**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+1.Install git bash
+2.Install Vscode, Sublime
+3.Web browser 
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+1.To access cloned directory:
+
+```sh
+  cd your-folder
+  git clone https://github.com/abel12-pro/Hello-Microverse.git
+```
+2.Run with live server 
+
+### Install
+
+To install linters and other project's dependencies run:
+
+
+```sh
+  cd your-folder
+  npm install
+```
+
+
+### Usage
+
+To run the project, execute the following command:
+   open index.html using live server or web browser 
+
+### Run tests
+
+To run tests, run the following command:
+
+1.For tracking linter errors in HTML files run:
+```sh
+npx hint .
+
+```
+2.For tracking linter errors in CSS or SASS files run:
+```sh
+npx stylelint "\*_/_.{css,scss}"
+
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Abel Fikre**
+
+
+- GitHub: [@abel12-pro](https://github.com/abel12-pro)
+- Twitter: [@abelfikre2](https://twitter.com/@abelfikre21)
+- LinkedIn: [abel-fikre-03b173a7](https://www.linkedin.com/in/abel-fikre-03b173a7/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[update the look and feel]**
+- [ ] **[media queries to make the site responsive]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/abel12-pro/H/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project just share to your families and give me like.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
```

Comment:
please, do not use comment in your readme file.Remove <!-- ACKNOWLEDGEMENTS--> comment from your readme file.

Make changes asper comment and send to the reviewers. 
Thanks.